pkg/vsphere/toolbox/service_test.go

```diff
@@ -288,6 +298,17 @@ func TestServiceRunESX(t *testing.T) {
 		}
 	}
 
+	if *testOn == string(types.VirtualMachinePowerStatePoweredOff) {
+		wg.Add(1)
```

Comment:
are we assured of receiving a power on event irrespective of any timing issues if the VM wasn't powered on initially? In the main path there's a timeout but not in the test.